lib/Net/DRI/DRD/DENIC.pm

```diff
@@ -171,5 +173,19 @@ sub domain_delete_authinfo
  return $reg->process('domain', 'delete_authinfo', [$dom, $rd]);
 }
 
+sub domain_restore
+{
+ my ($self, $reg, $dom, $rd) = @_;
+ return $reg->process('domain', 'restore', [$dom, $rd]);
+}
+
+# no need to create a new object (not even listed in their XSD only on their technical documentation)
+# RegAcc INFO request you can query your own public registrar contact details or those of others
+sub regacc_info
```

Comment:
I wonder if its worth using registrar_info (like eurid, afilias, and I think ZA?). So in the DENIC module you can add the comment regacc, but the function itself would be named registrar_info to be the same as other registries?

Reply:
Why not. Do you agree with this approach?
- https://github.com/thedarkwinter/Net-DRI/pull/343/commits/61fcc7e61d31e3d5cf59b0904dc7d9c5f4349c51

Reply:
yip thanks fine